state/filenames.go

```diff
@@ -15,5 +15,7 @@
 package state
 
 const (
-	ClusterFile = "cluster.yaml"
+	ClusterJsonFile = "cluster.json"
+	ClusterJsonPath = "/etc/kubicorn/"
```

Comment:
Sorry I just noticed this.. it looks like we are hard coding all JSON files to `/etc/kubicorn`.. I am wondering if we can't follow suit with the `fs` implementation and have the value here be optionally overwritten at runtime. 

There are command line flags, and environmental variables that can be set that will drop a cluster off here if needed. Seems like a more flexible approach than hard coding.. What are your thoughts?